features/fixtures/maze_runner/Assets/Scripts/Scenarios/Csharp/Config/MaxStringValueLength.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Collections.Generic;
+using BugsnagUnity;
+
+public class MaxStringValueLength : Scenario
+{
+
+    private string _oneHundredCharacters = "1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890";
+
+    public override void PrepareConfig(string apiKey, string host)
+    {
+        base.PrepareConfig(apiKey, host);
+        Configuration.MaxStringValueLength = 20;
+        for (int i = 0; i < 10000; i++)
+        {
+            Configuration.AddMetadata("test", "key-" + i.ToString(), _oneHundredCharacters);
+        }
+        Configuration.AddMetadata("test", "stringArray", new string[] { _oneHundredCharacters });
+        Configuration.AddMetadata("test", "stringList", new List<string>() { _oneHundredCharacters });
```

Comment:
I think it would be good to add a test for the `Dictionary` and `JsonArray` types here - any reason not to?

Reply:
Good idea, in doing so i realised that i had forgotten to add support for the Dictionary<string,string> type 👍